### ISSUE TITLE
LCL008 not available eventually breaking the effects from the Hue app

### ISSUE THREAD

**Issue by pyesandcakes** [diyhue/diyHue#1053]
### Issue does not already exist?

I have searched and found no existing issue

### Select Environment

Docker

### Home Assistant related?

No

### Description

I have the Hue Lighstrip Solo aka LCL008. Currently, in Diyhue I have it set under LST002 as that is the closest option. When trying to run the effects from the Hue app like the candle or fireplace it does not work. Not only does it not work but even the colors are not matched. Now zigbee2mqtt works fine and has the effects that work (except I can't find the speed option). I hope it will be added soon.

### Errorlog:

```
This can be downloaded from the webUI.
DiyHue > Bridge > Brdige Control > Download debug > Log tar
This is also included in Full debug
If diyHue does not start upload from normal log
```

### Steps to reproduce

By trying to run effect from the Hue app for the Lightstrip Solo.

### Please enter your operating system details here

If diyHue does not star use "uname -mrs"
Hue-Emulator Version:
Architecture:
OS:
Os version:
Os release:
Hardware:

### Upload debug tar

```
This can be downloaded from the webUI.
DiyHue > Bridge > Brdige Control > Download debug > Full debug
Do not upload from Force config dump, Force config dump contains sensitive information
``


### What DiyHue version(branch) are you using?

master (latest)

**Comment by hendriksen-mark** [comment]
Hi, the effects are not send to z2m, this can be made and will be in the next release of diyhue.
What do you meen with the colors not match?
If you put the color on red, green or blue is the color on the light not the same as where where you put it?

**Comment by pyesandcakes** [comment]
I mean when I activate an effect in the hue app the effect also has a set color and that color ( for example I activate a red candle and the actual light will just be blue or white without any effect) isn’t displayed.

Kind regards
Bruno Šimunić

**Comment by hendriksen-mark** [comment]
Yes that can happen, when the effect is activated there is also some other data transferred, like i said the effect is not transferred to z2m, i will make this in diyhue in the next days when i have some time to test this.

**Comment by pyesandcakes** [comment]
Sure, thanks for the quick reply.

Kind regards
Bruno Šimunić

**Comment by hendriksen-mark** [comment]
Hi, i added the effects to mqtt, please update and give feedback.

**Comment by pyesandcakes** [comment]
Great, I’ll be back home tonight so I’ll send feedback.

Kind regards
Bruno Šimunić

**Comment by pyesandcakes** [comment]
Effects work. But once I start an effect, for example, fireplace red, when I go to select the same effect but a different color it won't change. And it is missing all the new effects.

**Comment by hendriksen-mark** [comment]
yes that is a limitation of z2m i cannot find in the docs how to set color to the effects.
https://www.zigbee2mqtt.io/devices/8718699703424.html#effect-enum
request from the hue app:
```
{
    "on": {
        "on": True
    },
    "dimming": {
        "brightness": 84.0
    },
    "effects_v2": {
        "action": {
            "effect": "fire",
            "parameters": {
                "color": {
                    "xy": {
                        "x": 0.5799999999999998,
                        "y": 0.38
                    }
                },
                "speed": 0.5
            }
        }
    }
}
```

**Comment by pyesandcakes** [comment]
No problem. Thanks for the help.
